src/App/tests/testManagement.py

```diff
@@ -18,21 +20,39 @@ def test_Management__Navigation__manage_page_header__1(self):
 
     def test_Management__Navigation__manage_page_header__2(self):
         """It respects `zmi_additional_css_paths` ustring property."""
-        self.folder.manage_addProperty(
-            'zmi_additional_css_paths', '/foo/bar.css', 'ustring')
+        # unicode converters will go away with Zope 6
```

Comment:
I don't think there's a good reason why this test code uses `ustring`/`ulines` properties at all. I suggest you convert it to `string`/`lines` instead.

Reply:
Good catch!